organize/forms.py

```diff
@@ -0,0 +1,79 @@
+from django import forms
+from django_countries import countries
+from django_countries.fields import LazyTypedChoiceField
+from django_date_extensions.fields import ApproximateDateFormField
+
+
+from core.models import Event
+from core.validators import validate_approximatedate
+from .models import INVOLVEMENT_CHOICES
+
+PREVIOUS_ORGANIZER_CHOICES = (
+    (True, "Yes, I organized Django Girls"),
+    (False, "No, it’s my first time organizing Django Girls"))
+
+
+class PreviousEventForm(forms.Form):
+    has_organized_before = forms.ChoiceField(
```

Comment:
Using `forms.TypedChoiceField(coerce=bool)` here should work (and would allow us to clean up the weird `if has_organized_before == 'True'` in `clean()` later.